### ISSUE TITLE
Selection event setMinMax is not supported for selection type Cuboid

### ISSUE THREAD

**Issue by mikroskeem** [mikroskeem/WorldEditCUI#13]
```
[22:08:48] [main/INFO]: [STDERR]: com.mumfrey.worldeditcui.exceptions.InvalidSelectionTypeException: Selection event setMinMax is not supported for selection type Cuboid
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.render.region.Region.setMinMax(Region.java:108)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.cui.CUIEventBounds.raise(CUIEventBounds.java:40)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.CUIEventDispatcher.raiseEvent(CUIEventDispatcher.java:59)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.listeners.CUIListenerChannel.onMessage(CUIListenerChannel.java:34)
[22:08:48] [main/INFO]: [STDERR]: 	at eu.mikroskeem.worldeditcui.FabricModWorldEditCUI.lambda$onPluginMessage$0(FabricModWorldEditCUI.java:130)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_18859(class_1255.java:173)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_4093.method_18859(class_4093.java:23)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_16075(class_1255.java:135)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_5383(class_1255.java:115)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:956)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:619)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:204)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:193)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)
[22:08:48] [main/INFO]: [STDERR]: com.mumfrey.worldeditcui.exceptions.InvalidSelectionTypeException: Selection event setMinMax is not supported for selection type Cuboid
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.render.region.Region.setMinMax(Region.java:108)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.cui.CUIEventBounds.raise(CUIEventBounds.java:40)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.CUIEventDispatcher.raiseEvent(CUIEventDispatcher.java:59)
[22:08:48] [main/INFO]: [STDERR]: 	at com.mumfrey.worldeditcui.event.listeners.CUIListenerChannel.onMessage(CUIListenerChannel.java:34)
[22:08:48] [main/INFO]: [STDERR]: 	at eu.mikroskeem.worldeditcui.FabricModWorldEditCUI.lambda$onPluginMessage$0(FabricModWorldEditCUI.java:130)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_18859(class_1255.java:173)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_4093.method_18859(class_4093.java:23)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_16075(class_1255.java:135)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_1255.method_5383(class_1255.java:115)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:956)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:619)
[22:08:48] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:204)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:193)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:138)
[22:08:48] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[22:08:48] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[22:08:48] [main/INFO]: [STDERR]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by zml2008** [comment]
I expect this will be fixed by EngineHub/WorldEdit#1633

**Comment by zml2008** [comment]
Fixed in WE
